src/selectors.cc

```diff
@@ -290,7 +290,10 @@ find_opening(Iterator pos, const Container& container,
         if (nestable)
         {
             for (auto m : RegexIt{match[0].second, pos, container.begin(), container.end(), closing})
```

Comment:
I'd rather disable them with an attribute, can you use `for (auto m [[gnu::unused]] : ...`